pcap.c

```diff
@@ -303,6 +303,26 @@ pcap_next_ex(pcap_t *p, struct pcap_pkthdr **pkt_header,
 	return (p->read_op(p, 1, p->oneshot_callback, (u_char *)&s));
 }
 
+const u_char *
+pcap_next_with_file_offset(pcap_t *p, struct pcap_pkthdr *pkt_header,
```

Comment:
This is a version of `pcap_next()` with additional information, and inherit's `pcap_next()`s broken-by-design problems.  Don't bother with it.